docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+
+If you want to experiment with different scenarios how to proceed, go to the "Debug Console" where you can execute 
+code. For example, let's inspect the Pandas Dataframe by typing `main_df.info`.  
+You see that there are some empty rows at the end which don't contain useful data. The next two lines of code will 
+eliminate them. Click on "Step Over" two times, or set a new break point two lines further down and click on "Continue".
+Now, type again `main_df.info` in the Debug Console. You will see that the empty rows are gone.  
+
+![inspect dataframe](assets/inspect-dataframe.png)
+
+
+You see that the debugging mode is a useful tool to understand code and to inspect it for correctness. 
+
+| **Tip**                                                                                                   |
+|-----------------------------------------------------------------------------------------------------------|
+| **Make regular use of the debugging mode to check if your code really does what you think it should do!** | 
+
+
+## The benefits of version control
+One of the big benefits of version control is the diff viewer. Visual Studio highlights the changes you have introduced 
+since your last commit. 
+
+- Deletions are shown as red triangle.
+- Additions are shown as green bars.
+- Changed lines are shown as striped bars.
+
+Click on these visual elements to see a small popup that shows you the difference. In the popup, you can stage the 
+change, revert it, or jump to the next/previous change.
+
+![diffs](assets/diffs.png)
+
+Once you have a bunch of code changes that can be meaningfully grouped together, you should make a commit (and perhaps
+push it to a GitHub repo).
+
+
+| **Tips**                                                                |
+|-------------------------------------------------------------------------|
+| **Test your code (e.g. with the debugging mode) before committing it.** | 
+| **Make small commits that contain only one new feature.**               | 
+
+
+## Some extras
+### Data cleaning with OpenRefine
+[**OpenRefine**](https://openrefine.org/) is a tool for working with messy data. Once downloaded and installed, it runs 
+as a local server, accessed by your browser. So, all data remains on your own machine. Installation is quick and 
+painless: 
+```
+brew install openrefine
+```
+
+The potentials for the everyday work of the Client Services at DaSCH are twofold:
+1. Data cleaning (recommended): For this purpose, you can think of OpenRefine as a much better version of Excel. You 
+   can perform operations which would be very tiresome in Excel.
+2. Conversion to our dps-customised xml format for bulk upload (not recommended)
+
+Read more in [this report](https://docs.google.com/document/d/1Y_hZV8UV-Irw-7PLdhm0BGKnGfXJs8JO).
+
+
+### Git GUIs
+Git can be complicated, so you will appreciate to work with one of these GUIs:
+
+ - [**SmartGit**](https://www.syntevo.com/smartgit/) has a free edition. 
```

Comment:
SmartGit is good for power users, but anyone who's not experienced with Git tends to find it confusing. I'm not sure I'd recommend it in that context